dev-client/src/screens/LocationScreens/LocationDashboardContent.tsx

```diff
@@ -140,6 +139,9 @@ export const LocationDashboardContent = ({
       : state.project.projects[site.projectId],
   );
 
+  const soilIdData = useSelector(selectSoilIdData());
```

Comment:
Question: I believe this should be fine as-is, but do you think we should get in the habit of preferring `useSoilIdData` instead of some places using the selector directly? Just feels like doing this requires more thought about when we'd need to request potentially new data?

Reply:
I can give this some more thought - I agree with your point that this potentially requires more reasoning about when data gets loaded.

Reply:
Will refactor to use this.